lib/flight_scheduler/event_processor.rb

```diff
@@ -50,6 +50,7 @@ def cancel_job(job)
     Async.logger.info("Canceling job #{job.id}")
     return unless %w(PENDING RUNNING).include?(job.state)
 
+    # NOTE: Is the above and below state lines correct? Needs review
```

Comment:
I can across this section whilst refactoring but couldn't determine what it was meant to do. Two things:

1. Is line 54 meant to be `job.state = CANCELLED`? ATM it is doing an equality check but not using the result.
2. What is the purpose of line 55 `return if <pending>` considering we do a similar check above?

Reply:
Job cancellation has seen some refactoring in #8.  FWIW: line 54 should have been an assignment.  Line 51, prevents cancelling completed jobs and line 55 prevents attempting to kill processes for pending jobs (obviously the processes don't exist).

Reply:
Removed the comment in light of your changes:
https://github.com/openflighthpc/flight-scheduler-controller/pull/7/commits/06e21a3335ad12454e58b36fdb028b60f5378fd9